addons/shortcuts/shortcuts.lua

```diff
@@ -117,10 +117,16 @@ default_aliases = {
     cw5="Curing Waltz V",
     hw="Healing Waltz"
 }
+default_settings = {
+    include_items = false,
+}
 
-aliases = config.load('data\\aliases.xml',default_aliases)
+aliases = config.load('data/aliases.xml', default_aliases)
+settings = config.load('data/settings.xml', default_settings)
```

Comment:
Adding another settings file just to allow people to disable compatibility with items seems like a lot of work for no obvious benefit. I would just make it an always-on feature. No one is going to accidentally type `//proether+3` or `//vileelixir+1`

Reply:
I did this because of vague concerns that this would cause problems. https://discord.com/channels/338590234235371531/501099842097905675/883160617706483712

Reply:
This adds complexity without benefit, so I would eliminate it. Make items always-on and don't add a settings file.